rabbit/spring-cloud-starter-stream-sink-rabbit/src/main/java/org/springframework/cloud/stream/app/rabbit/sink/RabbitSinkConfiguration.java

```diff
@@ -86,7 +86,7 @@ public RabbitTemplate rabbitTemplate(ConnectionFactory rabbitConnectionFactory)
 	}
 
 	@Bean
-	@ConditionalOnProperty(name = "converterBeanName", havingValue = RabbitSinkProperties.JSON_CONVERTER)
+	@ConditionalOnProperty(name = "rabbit.converterBeanName", havingValue = RabbitSinkProperties.JSON_CONVERTER)
```

Comment:
This kind of stuff is done all over the place in boot's 'common application properties, e.g `server.compression.excluded-user-agents` so it shouldn't be an issue.  See http://docs.spring.io/spring-boot/docs/current/reference/html/common-application-properties.html